flocker/control/httpapi.py

```diff
@@ -53,18 +57,45 @@ def version(self):
         """
         return {u"flocker":  __version__}
 
+    # # Will we also have a /state endpoint? Or just route directly to the datasets path?
+    # @app.route("/state/datasets", methods=['GET'])
+    # @user_documentation("""
+    #     Get current cluster datasets
+    #     """, examples=[u"get state datasets"])
+    # @structured(
+    #     inputSchema={},
+    #     outputSchema={'$ref': '/v1/endpoints.json#/definitions/state/datasets'},
+    #     schema_store=SCHEMAS
+    # )
+    # def datasets(self):
+    #     """
+    #     Return all datasets in the cluster.
```

Comment:
Make sure you're very clear everywhere about whether your're talking about configuration or, as in this case, actual state.